bears/python/PEP8NotebookBear.py

```diff
@@ -84,13 +85,17 @@ def run(self, filename, file,
         will not change functionality of the code in any way.
 
         :param max_line_length:   Maximum number of characters for a line.
+                                  When set to 0 allows infinite line length.
```

Comment:
Make it in a passive mode, might put it as:-

> when set to 0, infinite line length is allowed 

Suggesting as per history :-)

Reply:
Well, if I do that, it will throw me an error (line_too_long). Besides, it follows PEP8Bear: https://github.com/coala/coala-bears/blob/master/bears/python/PEP8Bear.py#L36

So I think this is good enough for now.

Reply:
It is good enough for now. Passive mode would be better, but I am a very happy chappy to see this fix (especially because JSON is such a crappy format wrt line length)